tests/atest/rules/naming/settings-consistency/test_rule.py

```diff
@@ -0,0 +1,17 @@
+from tests.atest.utils import RuleAcceptance
+
+
```

Comment:
We have golden all task, golden all test, all test but task section, all task but test section but we don't have mixed tests (mixes test/task settings with task/test section).

Reply:
Added. Please check if it's fine now and if that's what you meant.